packages/react-dom/src/client/setAttribute.js

```diff
@@ -18,7 +19,11 @@ export function setAttribute(
   attributeName: string,
   attributeValue: string | TrustedValue,
 ) {
-  node.setAttribute(attributeName, (attributeValue: any));
+  if (attributeValue.$$typeof === REACT_OPAQUE_OBJECT_TYPE) {
```

Comment:
This works for updates, but need to handle similar case in `setInitialDOMProperties`, which is called during initial render.

Reply:
I think `node.setAttribute` is called during `setInitialDOMProperties` as well to set attributes:

`setInitialDOMProperties` -> `setValueForProperty` -> `setAttribute`/`setAttributeNs`